docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/admin-api-1x-docker-installation.md

```diff
@@ -0,0 +1,184 @@
+
+#### Admin API Application
+
+This service depends on the `pb-admin`  and subsequently `db-admin` services to run.
+
+**docker-compose.yml**
+
+```
+# ... above are other services
+adminapi:
+    build:
+    image: edfialliance/ods-admin-api:${ADMIN_API_TAG}
+    environment:
+      ADMIN_POSTGRES_HOST: pb-admin
+      POSTGRES_PORT: "${PGBOUNCER_LISTEN_PORT:-6432}"
+      POSTGRES_USER: "${POSTGRES_USER}"
+      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
+      DATABASEENGINE: "PostgreSql"
+      API_MODE: ${API_MODE}
+      AUTHORITY: ${AUTHORITY}
+      ISSUER_URL: ${ISSUER_URL}
+      SIGNING_KEY: ${SIGNING_KEY}
+      ADMIN_API_VIRTUAL_NAME: ${ADMIN_API_VIRTUAL_NAME:-adminapi}
+      API_INTERNAL_URL: ${API_INTERNAL_URL}
+    volumes:
+      - ../../Docker/ssl:/ssl/
+    depends_on:
+      - pb-admin
+    restart: always
+    hostname: ${ADMIN_API_VIRTUAL_NAME:-adminapi}
+    container_name: adminapi
+    healthcheck:
+      test: $$ADMIN_API_HEALTHCHECK_TEST
+      start_period: "60s"
+      retries: 3
+# ... below are network and volume configs
+```
+
+#### Admin API Database
+
+For the most part, the Admin API shares the same database schema as the Admin App. However, there are a few tables required for storing API client authentication which need to be initialized manually. You can see the details in [First-Time Configuration for Admin 1.x](../admin-api-1x-for-odsapi-5x-6x/first-time-configuration-for-admin-api-1x.md).
+
+Rather than introducing these tables explicitly, for Docker we have provided an alternative image for use with Admin API: [`edfialliance/ods-admin-api-db`](https://hub.docker.com/r/edfialliance/ods-admin-api-db), which is to be used **in place of** the existing `edfialliance/ods-api-db-admin` image for your DB service.
+
+**If you are introducing Admin API to an existing composition do** NOT **change the volume mapping configuration in order to preserve your data**.**** Only change the image and tag of the existing service. The below block is a sample of this, based on an example ODS / API Docker environment composition. Make sure you update the mode (`"``SharedInstance"`, `"YearSpecific"`, or `"DistrictSpecific"`) accordingly.
+
+**docker-compose.yml**
+
+```
+# ... above are other services
+db-admin:
+    image: edfialliance/ods-admin-api-db-admin:${ADMIN_API_DB_TAG}
+    environment:
+      POSTGRES_USER: "${POSTGRES_USER}"
+      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
+      API_MODE: <YOUR MODE HERE>
+    volumes:
+      - vol-db-admin:/var/lib/postgresql/data
+    restart: always
+    container_name: ed-fi-db-admin
+# ... below are other services
+```
+
+### .env Settings
+
+Add the following to your environment settings file to support Admin API. Note that when running both Admin App and Admin API, some of these settings may overlap. This is expected, and the same values can be used.
+
+**.env for Admin API**
+
+```
```

Comment:
Ideally, all code fences should have a language. We could try `ini` here. In the docusaurus config file, need to add 'ini' on line 177, because it is not one of the default languages.